lib/upfluence/utils/thrift.rb

```diff
@@ -1,2 +1,3 @@
 require 'upfluence/utils/thrift/middleware'
 require 'upfluence/utils/thrift/error_logger'
+require 'upfluence/utils/thrift/validation_error'
```

Comment:
could you put this file with a path reflecting the class  `'base/exceptions/validation_error'`